anvil/src/test/kotlin/dev/inkremental/XmlTest.kt

```diff
@@ -10,7 +10,7 @@ class XmlTest : Utils() {
     private var inflateCount = 0
     @Test
     fun testXml() {
-        Anvil.mount(container, Renderable {
+        Inkremental.mount(container, {
```

Comment:
Could you please move closing parenthesis in tests where `Renderable` was replaced with simple lambda like this?

Reply:
You mean to create trailing lambda?

Reply:
Yep.